src/Block/Adminhtml/Project/Tab/Categories.php

```diff
@@ -148,6 +160,7 @@ protected function _prepareColumns()
                 ]
             );
         }
+        $this->eventManager->dispatch('easytranslate_prepare_categories_columns');
```

Comment:
Don't we have to pass `$this` as an event param, so that we can actually add custom columns? Applies to all entities.